### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ check:
   - thoth-precommit
   - thoth-build
 build:
-  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py36-postgres:latest
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38-postgres:latest
```
Comment (bissenbay) [inline .aicoe-ci.yaml:5]: @fridex @harshad16 should I make this change?

Comment (fridex) [reply .aicoe-ci.yaml:5]: We don't have this image built. @harshad16 I don't remember where the Dockerfile was placed for it.

Comment (harshad16) [reply .aicoe-ci.yaml:5]: we manage the image Dockerfile in this repository, will get the py38 image setup as well.

Comment (fridex) [reply .aicoe-ci.yaml:5]: Cool, thanks :+1:

**Comment by fridex** [comment]
I see @harshad16 did the job :+1: https://quay.io/repository/thoth-station/s2i-thoth-ubi8-py38-postgres?tab=tags Should we unhold and merge?

**Review by harshad16 — APPROVED** [review]
lgtm, thanks :100:

**Pull request merged**